tests/v3_performance_lab.py

```
@@ -0,0 +1,1286 @@
+    for invocation in observation["invocations"]:
+        for input_record in invocation["inputs"]:
+            if input_record["path"] == old_path:
+                input_record["bytes"] = len(content)
+                input_record["sha256"] = LAB._sha256_bytes(content)
+
+
+def _substitute_linker_identity(document: dict[str, Any]) -> None:
+    observation = document["results"][0]["compile"]["observation"]
+    linker = observation["linker"]
+    clang = Path(document["toolchain"]["path"])
+    completed = subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)